README.rst

```diff
@@ -19,8 +19,7 @@
 Nengo: Large-scale brain modelling in Python
 ********************************************
 
-.. image:: http://c431376.r76.cf2.rackcdn.com/71388/fninf-07-00048-r2/image_m/fninf-07-00048-g001.jpg
-  :alt: An illustration of the three principles of the NEF
+Nengo is a Python library for building and simulating large-scale neural models. Nengo can create sophisticated spiking or non-spiking neural simulations with sensible defaults in a few lines of code. Yet, Nengo is highly extensible and flexible. You can define your own neuron types and learning rules, get input directly from hardware, build and run deep neural networks, drive robots, and even simulate your model on a completely different neural simulator or neuromorphic hardware.
```

Comment:
I'll fix this in the merge, so not blocking, but even in RST docs we want to keep to around 80 characters per line in order to make future diffs easier to read (scanning a really long line for the change takes a while). Then also, since we're manually adding line breaks, I prefer to use [semantic line breaks](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) for even easier plaintext reading and diffing.

Reply:
Also, I did kind of like that image as an eye catcher for the README... any objections to me adding it back in? I know the link is broken so I'll put it somewhere that we control.

Reply:
nope no objection, i took it out because of the broken link.